week7_03.py

```diff
@@ -0,0 +1,185 @@
+class Message:
+    """A Message class. Manages the content of messages.
+
+    Args:
+        message_id (int) : id of the message.
+        message_body (str) : content of the message.
+        sender (str) : who send the message.
+        recipient (str) : who recipient the message.
+        read (bool) : bool to mark if the message is read.
+    """
+
+    def __init__(self, message_id: int, message_body: str, sender: str, recipient: str, read: bool = False):
```

Comment:
This __init__ is redundant, read about dataclass decorator and use it - 
https://docs.python.org/3/library/dataclasses.html